src/components/issues/TaskRequestForm.tsx

```diff
@@ -100,7 +100,7 @@ const TaskRequestForm: FC<ActionFormProps> = ({
     const handleSubmit = async (e: MouseEvent) => {
         e.preventDefault();
         setIsLoading(true);
-        await createTaskRequest(state);
+        if (createTaskRequest) await createTaskRequest(state);
```

Comment:
Nit: How createTaskRequest here is working??

Reply:
Gets passed in props